src/test/utils/phaseInterceptor.ts

```diff
@@ -340,7 +449,8 @@ export default class PhaseInterceptor {
 
   /**
    * m2m to set mode.
-   * @param phase - The phase to start.
+   * @param mode - The {@linkcode Mode} to set.
+   * @param args - Additional arguments to pass to the original method.
    */
   setMode(mode: Mode, ...args: any[]): Promise<void> {
```

Comment:
NIT: This might be a better fit?
```suggestion
  setMode(mode: Mode, ...args: unknown[]): Promise<void> {
```